src/export/InstanceExporter.ts

```diff
@@ -322,6 +316,28 @@ export class InstanceExporter implements Fishable {
     cleanResource(instanceDef);
     this.pkg.instances.push(instanceDef);
 
+    // should we add the instanceOf to meta.profile?
```

Comment:
I wonder if we should move this up a few lines so it is _before_ the call to `this.validateRequiredElements`.  My concern is that if a profile has `meta.profile 1..*` and the author did not explicitly set a `meta.profile`, then won't `validateRequiredElements` complain about the missing `meta.profile`?  If so, that would be confusing to authors since they would see `meta.profile` populated in the output.  It seems to me that this code would be at home on line 310, after the call to `this.setAssignedValues`.

Reply:
I agree, the automatically provided profile should help contribute to a minimum cardinality. I've added that in the latest change set.